protocol/x/sending/client/cli/sending_cli_test.go

```diff
@@ -50,17 +47,8 @@ func (s *SendingIntegrationTestSuite) SetupTest() {
 	// Generated from the above Mnemonic.
 	s.validatorAddress = constants.AliceAccAddress
 
-	appOptions := appoptions.NewFakeAppOptions()
-
 	// Configure test network.
-	s.cfg = network.DefaultConfig(&network.NetworkConfigOptions{
-		AppOptions: appOptions,
-		OnNewApp: func(val networktestutil.ValidatorI) {
-			// Disable the Bridge and Price daemons in the integration tests.
-			appOptions.Set(daemonflags.FlagPriceDaemonEnabled, false)
-			appOptions.Set(daemonflags.FlagBridgeDaemonEnabled, false)
-		},
-	})
+	s.cfg = network.DefaultConfig(nil)
```

Comment:
Even though we don't have to do the clean-up here should we still anyways since it is easier for all tests to _copy_ the same code for clean-up even if it is a no-op.

Reply:
good point, thank you!